src/ast/passes/codegen_llvm.cpp

```diff
@@ -45,6 +45,10 @@
 #include "types.h"
 #include "usdt.h"
 
+#define _BT_DISASM_TMPFILE ".temp_bt_disassemble"
```

Comment:
Could we put this in TMPDIR instead or CWD? I know we try to clean up, but it still feels wrong to write files to the random directory we happen to be running it.
- Security software might be monitoring for writes to certain directories. We wouldn't want to trigger it by just debugging bpftrace in that directory.
- Maybe the filesystem metadata matters to some people. We're modifying the last updated time on the current directory by doing this.

Reply:
Fair enough. Updated